ingestor/app.py

```diff
@@ -149,11 +148,3 @@ def store_landing_data(event):
     ridership_data = landing.get_ridership_data()
     landing.upload_to_s3(json.dumps(trip_metrics_data), json.dumps(ridership_data))
     landing.clear_cache()
-
```

Comment:
We could keep this around and disable it with a flag (if we intend to use it again)

Reply:
@rudiejd started https://github.com/transitmatters/yankee-shuttle-ingestor to process it seperately